bun/starter/README.md

```diff
@@ -45,4 +45,4 @@ Sample `200` Response:
 
 ## 🔒 Environment Variables
 
-No environment variables required.
+No environment variables required.
```

Comment:
Please revert this change

Reply:
Reset this file please, there is a whitespace change`git checkout origin/main bun/starter/README.md`